openmetadata-ui/src/main/resources/ui/src/components/Glossary/ImportGlossary/ImportGlossary.test.tsx

```diff
@@ -91,15 +91,23 @@ describe('Import Glossary', () => {
     const uploader = await screen.getByTestId('upload-file-widget');
     const uploadButton = await screen.getByTestId('upload-button');
     const stepper = await screen.getByTestId('stepper');
+    const cancelBtn = await screen.findByTestId('cancel-button');
 
     expect(breadcrumb).toBeInTheDocument();
     expect(title).toBeInTheDocument();
     expect(uploader).toBeInTheDocument();
     expect(uploadButton).toBeInTheDocument();
     expect(stepper).toBeInTheDocument();
+    expect(cancelBtn).toBeInTheDocument();
+
+    act(() => {
+      fireEvent.click(cancelBtn);
+    });
+
+    expect(mockPush).toHaveBeenCalledTimes(1);
   });
 
-  it('Import Should work', async () => {
+  it('Import functionality Should work flow should work properly', async () => {
```

Comment:
Should work properly is wag term we should write something specific instead.

Import csv file should render preview or something on those lines based on your tests

one more thing we cannot verify more than one thing in a test case